cleanlab_studio/studio/enrichment.py

```diff
@@ -399,6 +409,27 @@ def resume(self) -> JSONDict:
         latest_job = self._get_latest_job()
         return api.resume_enrichment_job(api_key=self._api_key, job_id=latest_job["id"])
 
+    def run_online(
+        self,
+        data: Union[pd.DataFrame, List[dict]],
+        options: EnrichmentOptions,
+        new_column_name: str,
+    ) -> Dict[str, Any]:
+        """
+        Enrich data in real-time using the same logic as the run() method, but client-side.
+
+        Args:
+            data (Union[pd.DataFrame, List[dict]]): The dataset to enrich.
+            options (EnrichmentOptions): Options for enriching the dataset.
+            new_column_name (str): The name of the new column to store the results.
+
+        Returns:
+            Dict[str, Any]: A dictionary containing information about the enrichment job and the enriched dataset.
+        """
+        run_online = _get_run_online()
+        job_info = run_online(data, options, new_column_name, self._api_key)
```

Comment:
I don't think passing in `self._api_key` because `run_online` expects a `Studio` object?